lib/double-fetch-detection.c

```diff
@@ -0,0 +1,34 @@
+#include "linux/double-fetch-detection.h"
```

Comment:
To fix-up your new files' formatting just run clang-format on them. The kernel has a .clang-format style file that's pretty good to get started.

Reply:
Some context: https://clang.llvm.org/docs/ClangFormat.html

You probably have clang-format in your system's package manager.